src/plugins/curator/components/gallery-picker-field/index.js

```diff
@@ -1,11 +1,28 @@
+import React, { useMemo } from 'react';
 import SearchGalleriesSort from '@triniti/schemas/triniti/curator/enums/SearchGalleriesSort.js';
 import withRequest from '@triniti/cms/plugins/pbjx/components/with-request/index.js';
 import NodePickerField from '@triniti/cms/plugins/ncr/components/node-picker-field/index.js';
 
-export default withRequest(NodePickerField, 'triniti:curator:request:search-galleries-request', {
-  channel: 'picker',
-  initialData: {
-    sort: SearchGalleriesSort.ORDER_DATE_DESC.getValue(),
-    autocomplete: true,
-  }
-});
+const GalleryPickerFieldWithStatus = (props) => {
+  const { statuses, ...otherProps } = props;
+
+  const GalleryPickerField = useMemo(() => {
+    const initialData = {
+      sort: SearchGalleriesSort.ORDER_DATE_DESC.getValue(),
+      autocomplete: true
+    };
+    
+    if (statuses) {
+      initialData.statuses = statuses;
+    }
+    
+    return withRequest(NodePickerField, 'triniti:curator:request:search-galleries-request', {
+      channel: 'picker',
+      initialData
+    });
+  }, [JSON.stringify(statuses)]);
```

Comment:
@meghedi are you sure you need to stringify here? the statuses that are passed in should have consistent references